.circleci/config.yml

```diff
@@ -129,79 +149,94 @@ jobs:
 
   # current version deploy for previews
   website_deploy_preview:
-    <<: *defaults
+    <<: *default
+    executor: nodejs
     steps:
+      - *checkout
+      - *restore_node_modules
       - attach_workspace:
           at: ~/welcome-ui
-      - restore_cache:
-          key: << pipeline.parameters.cache_version >>-welcome-ui-{{ .Environment.CIRCLE_SHA1 }}
-      - run: node_modules/.bin/netlify deploy --filter=website --dir=website/out --alias $CIRCLE_BRANCH
+      - aws-cli/setup:
+          aws_access_key_id: AWS_ACCESS_KEY_ID
+          aws_secret_access_key: AWS_SECRET_ACCESS_KEY
+          region: ${AWS_REGION}
+      - run:
+          name: Process branch name
+          command: |
+            echo "export PROCESSED_BRANCH=$(echo "$CIRCLE_BRANCH" | sed "s/[^[:alnum:]-]/-/g" | tr '[:upper:]' '[:lower:]' | cut -c -63)" >> $BASH_ENV
+      - aws-s3/sync:
+          from: website/out
+          to: s3://welcome-ui/preprod/${PROCESSED_BRANCH}
```

Comment:
Is it possible to replace `preprod` by `preview` on the URL of the website? 👀

Reply:
Hello, yes it's possible but:
- we need to change the record (host)
- since we change the host, we need to change the lambda behind the dynamic websites
- we also need to change the path of the bucket

What's bother me a little is that for all preprod stage we are using `preprod` in our records, and it means changing the logic only for welcome-ui 😄
But I don't have a strong opinion, it's not critical ! What do you think @pauldn-wttj ?

Reply:
That's up to you @mleralec.

`welcome-ui` is not exactly following the same lifecycle as our other applications since it's an open source design system so I understand we don't have to have the rules. If you prefer to have `preview` instead of `preprod`, I'd say now is the time to make the switch. :slightly_smiling_face:

Reply:
Thanks for your answers @ClemWttJ @pauldn-wttj !
cc @theo-mesnil what do you think about that?

Reply:
I prefer preview because is not really a preprod :) if it's possible ❤️

Reply:
Ok on it then 😄

Reply:
Good for us :+1: